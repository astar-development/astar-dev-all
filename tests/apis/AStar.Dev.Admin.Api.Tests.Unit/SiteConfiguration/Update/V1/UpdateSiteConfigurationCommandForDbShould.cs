using AStar.Dev.Admin.Api.SiteConfigurations.Update.V1;
using AStar.Dev.Api.Usage.Sdk.Metrics;
using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Admin.Api.SiteConfiguration.Update.V1;

[TestSubject(typeof(UpdateSiteConfigurationCommandForDb))]
public sealed class UpdateSiteConfigurationCommandForDbShould
{
    [Fact]
    public void ContainTheExpectedValues()
    {
        var sut = new UpdateSiteConfigurationCommandForDb("mock-slug", new(), new());

        sut.ToJson().ShouldMatchApproved();
    }

    [Fact]
    public void ImplementIEndpointNameInterface()
    {
        var sut = new UpdateSiteConfigurationCommandForDb("mock-slug", new(), new());

        sut.ShouldBeAssignableTo<IEndpointName>();
    }
}
