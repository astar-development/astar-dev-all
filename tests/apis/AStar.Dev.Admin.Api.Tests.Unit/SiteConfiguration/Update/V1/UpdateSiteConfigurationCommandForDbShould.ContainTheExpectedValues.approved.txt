{
  "slug": "mock-slug",
  "baseUrl": "",
  "loginUrl": "",
  "loginEmailAddress": "",
  "username": "",
  "password": "",
  "updatedBy": "me"
}